src/test/interface/multi_chain_test.cpp

```diff
@@ -0,0 +1,66 @@
+#include <cmdstan/stansummary_helper.hpp>
+#include <stan/io/stan_csv_reader.hpp>
+#include <stan/services/error_codes.hpp>
+#include <stan/mcmc/chains.hpp>
+#include <test/utility.hpp>
+#include <gtest/gtest.h>
+
+TEST(interface, output_multi) {
+  std::vector<std::string> model_path;
+  model_path.push_back("src");
+  model_path.push_back("test");
+  model_path.push_back("test-models");
+  model_path.push_back("test_model");
+
+  std::string command
+      = cmdstan::test::convert_model_path(model_path)
+        + " sample num_warmup=200 num_samples=1 num_chains=2 random seed=1234"
+        + " output file=" + cmdstan::test::convert_model_path(model_path)
+        + ".csv";
+
```

Comment:
This test implicitly tests that the default init chain id is 1 (as we test that the filename ends with _1.csv)...do we want that? Or is it better to specify chain_id=1 as an argument to cmdstan? I don't know the answer, but some thought and doc would be nice here.

Reply:
Yes this was wrong before good catch! I set this to `id=10` so that in the test we then look for files ending in `10.csv` and `11.csv`. I think I should add docs to the `output` `file` argument so that folks know that for `id=N` and `num_chains=j` the output will be `output_{N + 1}.csv`, `output_{N + 2}.csv`, ..., `output_{N + j}.csv`